torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1365,6 +1367,33 @@ at::Tensor XLANativeFunctions::expand(const at::Tensor& self,
       bridge::GetXlaTensor(self), torch::lazy::ToVector<int64_t>(size)));
 }
 
+at::Tensor XLANativeFunctions::expand(const at::Tensor& self,
+                                      c10::SymIntArrayRef size, bool implicit) {
+  XLA_FN_COUNTER("xla::");
+  std::vector<c10::SymInt> _sizes = torch::lazy::ToVector<c10::SymInt>(size);
+  std::vector<torch::lazy::NodePtr> size_nodes;
+  std::vector<int64_t> upper_bounds;
+  std::vector<bool> dynamic_dims;
+  /* TODO: move this code to a helper function */
+  for (auto& _size : _sizes) {
+    std::shared_ptr<c10::SymbolicIntNode> _symbolicIntNode =
+        _size.toSymbolicIntNode();
+    auto _lazySymIntNode =
+        std::dynamic_pointer_cast<torch::lazy::SymbolicIntNode>(
+            _symbolicIntNode);
+    auto size_node = _lazySymIntNode->node_;
+    size_nodes.push_back(size_node);
+    upper_bounds.push_back(
+        std::dynamic_pointer_cast<torch_xla::DimensionNode>(size_node)
+            ->getStaticValue());
+    dynamic_dims.push_back(
+        std::dynamic_pointer_cast<torch_xla::DimensionNode>(size_node)
+            ->isDynamic());
```

Comment:
how close r you to testing this?

Reply:
Pretty close. We are resolving an internal build issue for the tester. I hope I can do it by tomorrow.